src/main/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporter.java

```diff
@@ -467,6 +481,23 @@ private String getIdentityProvider(Event event) {
         return identityProvider;
     }
 
+    /**
+     * Retrieve the real realm name in the event by id from the RealmProvider.
+     *
+     * @param realmId Id of Realm
+     * @param realmProvider RealmProvider instance
+     * @return Realm name
+     */
+    private String getRealmName(String realmId, RealmProvider realmProvider) {
+        RealmModel realm = null;
+        if (realmId != null) {
+             realm = realmProvider.getRealm(realmId);
+        }
+        if (realm != null) {
+            return realm.getName();
+        }
+        return null;
```

Comment:
Is the realm provider available on older Keycloak versions as well? Should we at least return the `realmId` if we're not able to locate the name here?